package.json

```diff
@@ -4,14 +4,14 @@
   "description": "A readable stream of CloudFormation stack events",
   "main": "index.js",
   "scripts": {
-    "test": "tap test.js"
+    "test": "tape test/*.test.js"
   },
   "author": "John Firebaugh <john@mapbox.com>",
   "license": "ISC",
   "dependencies": {
-    "aws-sdk": "^2.0.0-rc13"
+    "aws-sdk": "^2.2.0"
   },
   "devDependencies": {
-    "tap": "^0.4.8"
+    "tape": "~4.4.0"
```

Comment:
I'm not sure why but `tap` gave me strange timeout errors with these tests. I switched to `tape` for this reason and no other.